tests/functional/u/use/used_before_assignment_issue85.py

```diff
@@ -0,0 +1,8 @@
+# pylint: disable=missing-module-docstring, missing-function-docstring
```

Comment:
Could you maybe add a link to the issue here? And perhaps even a docstring explaining that we are assuming the try failed? That might be helpful for future reference.